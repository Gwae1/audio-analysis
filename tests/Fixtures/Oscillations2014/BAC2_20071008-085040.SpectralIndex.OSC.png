version https://git-lfs.github.com/spec/v1
oid sha256:3aea9968809c69b0bac868a6f530db0bb72cc906863a817be29b47963d7a8ba7
size 1549
