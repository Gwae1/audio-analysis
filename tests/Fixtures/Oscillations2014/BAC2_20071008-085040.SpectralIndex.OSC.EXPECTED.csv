version https://git-lfs.github.com/spec/v1
oid sha256:85c6ddad05a8d85886bc6723106a2a08f7f505da38a153c1939922e276849284
size 1052
